sdk/notificationhubs/Azure.ResourceManager.NotificationHubs/src/Generated/NotificationHubNamespaceData.cs

```diff
@@ -78,19 +78,19 @@ internal NamespaceResourceData(ResourceIdentifier id, string name, ResourceType
         /// <summary> The time the namespace was updated. </summary>
         public DateTimeOffset? UpdatedOn { get; set; }
         /// <summary> Endpoint you can use to perform NotificationHub operations. </summary>
-        public string ServiceBusEndpoint { get; set; }
+        public Uri ServiceBusEndpoint { get; set; }
         /// <summary> The Id of the Azure subscription associated with the namespace. </summary>
         public string SubscriptionId { get; set; }
```

Comment:
Should we change the type to `Guid`?

Reply:
We keep the SubscriptionId as string for all RPs